script/Arcueid/Day 8/08_00_ARC08_7_2.txt

```diff
@@ -186,67 +205,72 @@ I unleash the blade of my knife.
 [sha:a15d775282c3ac6418b8855e08f35c874db54b6d]{
 -- Page 482, Offset 20934.
 -- 眼鏡のつるに手をかける。
-I put my hand on the temple of my glasses.
+// Had to look this up but I guess people do call the long bit on the glasses the 'temple'?
```

Comment:
LMAO I had to google the exact same thing. I had never heard this before and originally had a comment on Manolo's PR about it.

I'm fine dropping it if neither of us has ever heard it before